### PR TITLE
Influxdb data loader

### PR DESCRIPTION
**Pull request by mazieb**
# Description
Add data loader for InfluxDB. 

# How Has This Been Tested?
- [x] Tested Locally

# Checklist
- [x] The PR is tagged with proper labels (bug, enhancement, feature, documentation)
- [x] I have performed a self-review of my own code
- [x] I have added unit tests that prove my fix is effective or that my feature works
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

cc:
@wangxiaoyou1993

### CODE REVIEW

**Comment by Anaisdg** [comment]
Thank you so much for this! 
This is so wonderful. :) 

However I also recommend checking out the [InfluxDB v3 Python Client Library](https://github.com/InfluxCommunity/influxdb3-python). 
It wraps the Apache Arrow Python Flight Client for querying from InfluxDB Cloud v3 (OSS will be available alter this year). 

The v2 client library you're using won't be compatible with v3.

**Comment by mazieb** [comment]
Thanks for the link to the new InfluxDB v3 library. I didn't know that yet. I think using the old library is sufficient for the moment, as the new library just [wraps the old one](https://github.com/InfluxCommunity/influxdb3-python/blob/adb95eff48b7ed7e3d999b3e012a42f461647b5a/influxdb_client_3/__init__.py#L3C39-L3C39) and doesn't offer a rich API. But the old library definitely need to be replaced at some time in the future.

**Comment by wangxiaoyou1993** [comment]
Hi @mazieb , is this PR ready for review?

**Comment by wangxiaoyou1993** [comment]
Could you rebase master branch? I think some changes have already been made in the other merged PR.

File: mage_ai/streaming/sources/influxdb.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+from influxdb_client import InfluxDBClient, QueryApi
+from influxdb_client.client.flux_table import TableList
+
+from mage_ai.shared.config import BaseConfig
+from mage_ai.streaming.constants import DEFAULT_BATCH_SIZE
+from mage_ai.streaming.sources.base import BaseSource
+
+
+def build_query(bucket: str, start_time: float, stop_time: float, time_delay: str):
+    '''
```
Comment (wangxiaoyou1993) [inline mage_ai/streaming/sources/influxdb.py:15]: nit: we use google docstring format https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

File: mage_ai/streaming/sources/influxdb.py [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+@dataclass
+class InfluxDbConfig(BaseConfig):
+    url: str
+    token: str
+    org: str
+    bucket: str = 'data'
+    measurement: str = 'default'
+    batch_size: int = DEFAULT_BATCH_SIZE
+    timeout_ms: int = 1000
+    time_delay: str = '-5m'
```
Comment (wangxiaoyou1993) [inline mage_ai/streaming/sources/influxdb.py:89]: Is there a reason to use negative number of time_delay?

**Review by wangxiaoyou1993 — APPROVED** [review]

**Comment by Anaisdg** [comment]
@mazieb yes we're actually recommending to users to not upgrade to 2.x and wait for 3.x because migration from 1.x to 3.x is easier.

**Pull request merged**